notebooks/2023-01-23 Differential abundance testing.Rmd

```diff
@@ -184,35 +218,43 @@ t.test.res.all %>%
 ```
 
 ```{r}
-dat <- prot_res_flt %>%
+dat <- prot_res_flt_adj %>%
   exprs()
 
 # Performing the equivalent of a one-sample t-test, so the design matrix is just an intercept
 design <- cbind(Intercept = rep(1, ncol(dat)))
 
 fit <- lmFit(dat, design)
-efit <- eBayes(fit)
+efit <- eBayes(fit, trend =TRUE)
```

Comment:
trend=TRUE will make sure to use the trend of the relationship between expression and variance, as far as it exists...